keyworker-specs/src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/specs/KeyworkerPrisonStatsSpecification.groovy

```diff
@@ -126,36 +126,6 @@ class KeyworkerPrisonStatsSpecification extends BrowserReportingSpec {
                 .withQueryParam("toDate", WireMock.equalTo(yesterday.toString())))
     }
 
-    def "should display an error if before date is after to date and disable submit button"() {
```

Comment:
The new version of react-datetime takes into account custom validate functions (which we have). This means that it considers the form invalid and disables the Update button if the from date is after the to date. This makes this test unnecessary and impossible.